toolchain/check/context.h

```diff
@@ -6,6 +6,7 @@
 #define CARBON_TOOLCHAIN_CHECK_CONTEXT_H_
 
 #include "common/map.h"
+#include "common/ostream.h"
```

Comment:
This is no longer used in this PR and can be removed.

Reply:
Removing